### PR TITLE
dind preset

### PR DESCRIPTION
**Pull request by BenTheElder**
this should simplify using it a bit and shrink our config
xref: https://github.com/kubernetes/test-infra/issues/8812

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -3031,21 +2882,13 @@ periodics:
...
-      env:
-      - name: DOCKER_IN_DOCKER_ENABLED
-        value: "true"
       # docker-in-docker needs privileged mode
       securityContext:
         privileged: true
-      volumeMounts:
-      - name: docker-graph
-        mountPath: /docker-graph
+      resources:
```
Comment (BenTheElder) [inline prow/config.yaml]: also fixed this incorrect resource request ...

**Comment by krzyzacy** [comment]
anything's using that in config/jobs?

**Comment by BenTheElder** [comment]
@krzyzacy I haven't migrated them yet, there were like 48 jobs in prow/config.yaml alone and I'm doing it by hand because currently it's hard to automatically refactor the main config..

**Comment by BenTheElder** [comment]
OK _ALL_ jobs have been migrated to the preset @krzyzacy

**Comment by BenTheElder** [comment]
/hold cancel
@ixdy :eyes:

**Pull request merged**